src/configuration/configuration_manager.cpp

```diff
@@ -0,0 +1,149 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// configuration_manager.cpp
+//
+// Identification: src/configuration/configuration_manager.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+#define __PELOTON_CONFIG__
+
+#include "common/exception.h"
+#include "configuration/configuration_manager.h"
+#include "configuration/configuration.h"
+#include "catalog/config_catalog.h"
+
+uint64_t GET_INT(const std::string& name) {
```

Comment:
Don't pollute the global namespace.
Make these static methods inside of a utility class.